bats_ai/core/models/recording.py

```diff
@@ -34,6 +47,7 @@ class Recording(TimeStampedModel, models.Model):
         Species, related_name='recording_official_species'
     )  # species that are detemrined by the owner or from annotations as official species list
     unusual_occurrences = models.TextField(blank=True, null=True)
+    tag = models.ForeignKey(RecordingTag, on_delete=models.DO_NOTHING, null=True)
```

Comment:
I'm guessing that first request we will get is being able to have multiple tags on a Recording instead of a single tag?
May want to confer with Aashish or USGS to determine if this is the case.

Reply:
After thinking about this some more, I've decided to add it now instead of later.